src/extension.ts

```diff
@@ -38,6 +38,7 @@ export class Extension {
      * available to the user. The credentials are also used
      * when making requests from the Vonage API.
      */
+    Auth.loginWithNeru();
```

Comment:
do we wanna check if the Auth is logged in already before attempting the neru one? might conflict if people log in with the UI and use neru